Description: Fix sorting to match what we see on Debian/Ubuntu builds.
 This might need another look sometime to figure out why this appears
 to work upstream, but not for us, but given hu_HU sorting was entirely
 wrong before 2.26 (apparently), this small discrepancy seems forgivable.
Author: Adam Conrad <adconrad@0c3.net>

--- glibc/localedata/hu_HU.in	2017-08-02 06:57:16.000000000 -0600
+++ glibc/localedata/hu_HU.in	2017-09-01 02:16:45.725379042 -0600
@@ -241,8 +241,8 @@
 alphabet ö              ; ö = ő (unless that's the only difference), but these come strictly after o and ó.
 alphabet ő
 alphabet öö
-alphabet öő
 alphabet őö
+alphabet öő
 alphabet őő
 alphabet őp
 alphabet öq
@@ -279,8 +279,8 @@
 alphabet űp
 alphabet üq
 alphabet üü
-alphabet üű
 alphabet űü
+alphabet üű
 alphabet űű
 alphabet v
 alphabet w
